gpt_index/data_structs/node_v2.py

```diff
@@ -91,6 +94,20 @@ def next_node_id(self) -> str:
             raise ValueError("Node does not have next node")
         return self.relationships[DocumentRelationship.NEXT]
 
+    @property
+    def parent_node_id(self) -> str:
+        """Parent node id."""
+        if DocumentRelationship.PARENT not in self.relationships:
+            raise ValueError("Node does not have parent node")
+        return self.relationships[DocumentRelationship.PARENT]
+
+    @property
+    def child_node_ids(self) -> List[str]:
+        """Child node ids."""
+        if DocumentRelationship.CHILD not in self.relationships:
```

Comment:
as a nice to have, can add a validation check that self.relationships[DocumentRelationship.CHILD] is a list of strings